index.md

```diff
@@ -11,9 +11,18 @@ like other GSA organizations in other departments, we intend to serve as both a
 social space, and as a democratic body that will advocate for the interests of
 Khoury graduate students with the college administration.
 
-<p><mark>Contact:</mark> <a href="mailto://kgsa-oc@googlegroups.com">kgsa-oc@googlegroups.com</a></p>
+If you want to lear more about KGSA, check out [this presentation of our agenda][gsa-slides-spring-2021]
+from Spring 2021.
 
-Or get in touch with any of the current organizing members:
+[gsa-slides-spring-2021]: https://docs.google.com/presentation/d/1prKtG2riNKsg6yTM3xEMUW1OXdo_X6J-0_xUjAd_d8w/edit?usp=sharing
+
+Contact us via [kgsa-oc@googlegroups.com](mailto://kgsa-oc@googlegroups.com) or
+say “Hi!” in the [Khoury PhDs Slack][khoury-slack] `#gsa` channel: we
+are always happy to meet new people and hope to get more students as active members!
+
+[khoury-slack]: https://khouryphds.slack.com
```

Comment:
Maybe a note that people can join the Slack with their `@husky.neu.edu` address?

Reply:
Yeah, I was hoping to defer this kind of issues implicitly suggesting that in case of anything not working they should contact us. But maybe you are right and Slack deserves this clarification. How about:

> To get added to the Slack, use the `<NEU-username>@husky.neu.edu` email.

?

Reply:
LGTM!

(Last week, someone actually emailed me about not being able to access the Slack.)

Reply:
Yeah, let's add this note about a husky email.